.github/workflows/build-devcontainer.yaml

```diff
@@ -41,17 +49,12 @@ jobs:
           file: .devcontainer/base-image/Dockerfile
           tags: ${{ steps.meta.outputs.tags }}
           platforms: linux/amd64, linux/arm64
-          push:
-            # If it's a push to main, or a tag, or explicitly called, push the image to the registry.
-            # Don't if it's on a schedule.
-            ${{ (github.ref == 'refs/heads/main' && github.event_name == 'push')
-            || startsWith(github.ref, 'refs/tags/') || github.event_name ==
-            'workflow_dispatch' }}
+          push: ${{ inputs.push == 'true' }}
```

Comment:
This can be?

```suggestion
          push: inputs.push
```

Reply:
I'm not sure — I get confused by these. Within an `if`, we don't need the brackets. But outside, it's possible that `'false'` is interpreted as a bool.

If you're confident, then commit — I've resorted to almost always doing the verbose `== 'true'` thing

Reply:
I see, thanks. Let's keep as is now.